tests/doghouse/test_delta_engine.py

```
@@ -0,0 +1,117 @@
+    
+    assert delta.head_changed
+    assert len(delta.added_blockers) == 1
+    assert delta.added_blockers[0].id == "2"
+    assert len(delta.removed_blockers) == 1
+    assert delta.removed_blockers[0].id == "1"
+    assert len(delta.still_open_blockers) == 0
+
+def test_compute_delta_empty_blockers():
+    engine = DeltaEngine()
+    baseline = Snapshot(
+        timestamp=datetime.datetime(2026, 1, 1, tzinfo=datetime.timezone.utc),
```

Ruff (0.15.7)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)